test/main.sh

```diff
@@ -228,11 +228,11 @@ if [ -n "${INACCESSIBLE_DIRS:-}" ]; then
 fi
 
 if [ "${LXD_TMPFS:-0}" = "1" ]; then
-  mount -t tmpfs tmpfs "${TEST_DIR}" -o mode=0751 -o size=7G
+  mount -t tmpfs tmpfs "${TEST_DIR}" -o mode=0751,noatime -o size=7G
```

Comment:
I think we should stick use relatime as it gives similar performance benefits than noatime but still updates atime for written files.